### ISSUE TITLE
run sometimes goes beyond event

### ISSUE THREAD

**Issue by alehander42** [mozilla/rr#2201]
the tutorial says in https://github.com/mozilla/rr/wiki/Usage

> If you want to restart replay at a different event, you can pass the event number as an argument to the gdb run command

however sometimes I can write `run 10` and get to the 11 event, but  a lot of stuff happens on 10 before that.

Is that normal, and if so , is there a better way than looping with smaller and smaller event arg to go to 10?

**Comment by rocallahan** [comment]
Yeah, `replay -g <N>` replays to the *end* of event N and you usually want to replay to the start. Unfortunately it's a bit difficult to fix so I haven't gotten around to it. I just try smaller event numbers, sorry :-(.

**Comment by alehander42** [comment]
no problem, I am using this  approach now
is it something that an outsider to the codebase has a chance to fix, or it requires more experience with it?

**Comment by rocallahan** [comment]
I think a motivated outsider could fix it. Start in `ReplayCommand.cc` :-)
